### PR TITLE
Add dark mode at system level

### PR DESCRIPTION
**Pull request by broth68**

### CODE REVIEW

**Comment by broth68** [comment]
Js change to do in article :

in dark mode generate :
background-image: radial-gradient(closest-side, rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.85)), url("/assets/img/182");

instead of :
background-image: radial-gradient(closest-side, rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.85)), url("/assets/img/182");

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+	--layout-block-box-shadow: black;
+	--question-border: darkgray;
+	--question-color: darkgray;
+	--categories-li-background-color: antiquewhite;
+	--categories-li-color: #4d4944;
+	--layout-nav-ul-border: black;
+	--table-background: ghostwhite;
+	--layout-nav-ul-background-color: rgba(248, 284, 255, .25);
+	--modal-content-box-shadow: rgba(0, 0, 0, .075);
+	--article-container-background-color: rgba(255, 255, 255, .35);
```
Comment (Pandraghon) [inline public/stylesheets/style.css:18]: Can you add the rules you put in the light @media that are missing here?
Can you also add 2 new variables for the article list background rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0.85) in light and rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.85) in dark? I will change the JS accordingly

**Review by Pandraghon — CHANGES_REQUESTED** [review]

**Comment by Pandraghon** [comment]
Thanks for your job.
Just one or two things left, and we're good :+1:

**Comment by Pandraghon** [comment]
Closes #13

**Comment by broth68** [comment]
Thanks for your review, I'll check that ASAP.

**Review by Pandraghon — APPROVED** [review]

**Pull request merged**